hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -241,6 +242,19 @@ public void testOMClientProxyProvider() {
         ozoneManager.getOmRpcServerAddr()));
   }
 
+  @Test
+  public void testVolumeSetOwner() throws IOException {
+    String volumeName = UUID.randomUUID().toString();
+    store.createVolume(volumeName);
+
+    String ownerName = "someRandomUser1";
+
+    ClientProtocol proxy = store.getClientProxy();
+    proxy.setVolumeOwner(volumeName, ownerName);
+    // Set owner again. Expect no NPEs nor timeouts
```

Comment:
Minor NIT: We don't need this comment, as this PR fixes the bug in the code :)

Reply:
yep my intention is to remind someone in the future in case this breaks again by some new code.

Reply:
Revised comment.